libraries/package-extractor/src/scripts/createLinks/utilities/constants.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import os from 'node:os';
+import path from 'node:path';
+import type { TARGET_ROOT_SCRIPT_RELATIVE_PATH_TEMPLATE_STRING as TargetRootScriptRelativePathTemplateString } from '../../../PackageExtractor';
+
+/**
+ * The maximum number of concurrent operations to perform.
+ */
+export const MAX_CONCURRENCY: number = os.cpus().length * 2;
```

Comment:
Seems high?

Reply:
It is, but it's because the operations that we are parallelizing generally shouldn't have too much overhead (ex. creating links is essentially synchronous). I've reduced it to the number of CPUs to ensure we still have a sizeable concurrency but avoid over-saturating the system in more demanding scenarios